src/transformers/convert_graph_to_onnx.py

```diff
@@ -337,11 +337,15 @@ def convert_tensorflow(nlp: Pipeline, opset: int, output: Path):
 
         # Forward
         nlp.model.predict(tokens.data)
-        onnx_model = convert_keras(nlp.model, nlp.model.name, target_opset=opset)
-        save_model(onnx_model, output.as_posix())
```

Comment:
With the new `tf2onnx` API, we no longer need a separate save model step - this is handled by the `output_path` arg in `tf2onnx.convert.from_keras()` below